datasets/lxmert_pretraining_beta/test_multi_array.py

```diff
@@ -0,0 +1,34 @@
+import nlp
```

Comment:
Feel free to convert this script to a unit test inside our `tests` folder.
We're using pytest for testing, you get it along with all the libraries we use for development (black, isort... ) with
```
pip install -e .[dev]
```